.github/workflows/validate.yaml

```diff
@@ -12,6 +12,6 @@ jobs:
       - name: Install preprequisites
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple opera
```

Comment:
Ok, yeah that's a problem - if we test the examples with production version of opera the new PRs and examples with new features will probably have to wait (a month or two) until we release a new version of opera which is kinda bad. But on the other hand some old examples could fail if we test with dev version of opera or there could be a bug in dev version which would cause failures. However I believe that the second scenario is still better because we can edit the examples or fix bugs in dev version of opera right away instead of waiting for production version. So, here I wanted to establish regular tests with dev version of opera and then in the future nightly tests with production version to ensure stability. But it could also be other way around. I guess that we will have to discuss this more and find a solution that would allow us having validated examples without production releases. For the start maybe just a manual CI job.

All in all, for now I have removed this dev version testing and went back to testing with regular production opera.